akka-actor-tests/src/test/scala/akka/actor/ActorRefSpec.scala

```diff
@@ -466,8 +466,8 @@ class ActorRefSpec extends AkkaSpec("""
         def receive = { case name: String => sender() ! context.child(name).isDefined }
       }), "parent")
 
-      assert(Await.result((parent ? "child"), timeout.duration) === true)
```

Comment:
`assert` is a macro and is not usable "as is", for the moment I switched to `require`, but I was expecting this to work out of the box, maybe @nicolasstucki or @smarter can shed a light?

Reply:
Since `assert` is a macro and Scala 3 rejects it.

Reply:
@nicolasstucki there's no actual issue, this project is just using a scala 2 version of scalatest instead of the scala 3 version: https://github.com/akka/akka/pull/29929#discussion_r551913737